ci/test/02_run_container.sh

```diff
@@ -71,6 +71,13 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
    fi # End temporary exclusion
   fi
 
+  if [ -n "$DANGER_CI_ON_HOST_BUILD_FOLDER" ] ; then
```

Comment:
Six lines of a simple conditional statement hardly constitute a "complexity explosion" :)

Since this functionality is currently needed for just one job, I’d prefer to leave it as is.

Reply:
> I'll probably fix it up in a follow-up if you leave it as-is for now.

Thank you! I'll be happy to review your PR.